### PR TITLE
Pedro/bal 388 depth cost chart

### PR DESCRIPTION
**Pull request by yvesfracari**
This PR adds:
- Fix variant tokens updates
- Add depth cost chart (on video)
- Define layout of graphView component (overflow and flex)

https://www.loom.com/share/5c5d3e4895db47458afe25054f175906

### CODE REVIEW

File: packages/math/src/stable/index.ts [hunk 1]
```diff
@@ -162,6 +162,45 @@ function _tokenInForExactSpotPriceAfterSwap(
...
+    StableMaths._spotPriceAfterSwapTokenInForExactTokenOut(outGuess, {
+      ...poolPairData,
+    });
+  const diffFromSpotPrice = spotPrice.minus(guessedSpotPrice);
+  if (diffFromSpotPrice.abs().lte(spotPricePrecision)) {
+    return outGuess;
+  }
+
+  iteration_number += 1;
+  if (iteration_number > 255) {
```
Comment (luizakp) [inline packages/math/src/stable/index.ts:187]: how did you define 255?

File: packages/next-app/src/app/stableswapsimulator/(components)/DepthCost.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+import { StableMath } from "@balancer-pool-metadata/math/src";
+import { PlotType } from "plotly.js";
+
+import Plot, { defaultAxisLayout } from "#/components/Plot";
+import { AnalysisData, useStableSwap } from "#/contexts/StableSwapContext";
+
+export default function DepthCost() {
+  const { indexAnalysisToken, baselineData, variantData } = useStableSwap();
+
+  const tokenY = baselineData?.tokens[indexAnalysisToken];
```
Comment (luizakp) [inline packages/next-app/src/app/stableswapsimulator/(components)/DepthCost.tsx:13]: hmmm I think we should keep the same convention for names and objects data to be consumed by the plot. If we decide this still we should refactor the other charts

**Review by luizakp — APPROVED** [review]
just left some comments and a open thought hehehe

File: packages/math/src/stable/index.ts (hunk 1 above)
Comment (yvesfracari) [reply packages/math/src/stable/index.ts:187]: It is the same number to calculate the invariant in the solidity

Comment (ribeirojose) [reply packages/math/src/stable/index.ts:187]: `#should` a comment here it just pointing that out 😬

**Pull request merged**